colir.el

```diff
@@ -37,8 +37,8 @@
   "Select a method to compose two color channels."
   :group 'ivy
   :type '(radio (function-item colir-compose-alpha)
-                (function-item colir-compose-overlay)
-                (function-item colir-compose-soft-light)))
+          (function-item colir-compose-overlay)
+          (function-item colir-compose-soft-light)))
```

Comment:
FWIW, I would advise using `common-lisp-indent-function` a bit more judiciously than this (if at all) to avoid such indentation, which IMO is both too ugly and too inconsistent with the rest of Emacs. I think the `common-lisp-indent-function-for-elisp` and `common-lisp-indent-function` properties aren't specified in a complete or consistent enough way to warrant going full `common-lisp-indent-function`.